frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTitles.tsx

```diff
@@ -125,6 +125,7 @@ const SoftwareTitles = ({
         teamId,
         addedSoftwareToken,
         ...vulnFilters,
+        ...(showVersions ? { without_vulnerability_details: true } : {}),
```

Comment:
Not blocking (I see this is the cherry pick to RC) but out of curiosity – why not something like `withoutVulnDetails: showVersions` ?

Reply:
I was hoping to exclude the entire query string parameter when it is not needed.

Reply:
Oh nice, right. Didn't read below to see that this is referencing the query key directly for the service method arguments.

Reply:
Sorry I probably should have gotten a ✅ from someone on front end on the original PR before merging. I will make sure to do that next time there are any react changes.

Reply:
Actually now that we're on the subject..`withoutVulnerabilityDetails: true` would be better, since [`getSoftwareVersions` uses `convertParamsToSnakeCase`](https://github.com/fleetdm/fleet/blob/874a1bf495d778a9ac6bc1c16e8034e5adb511ee/frontend/services/entities/software.ts#L230) to process all of these field names accordingly anyway.